build/build.go

```diff
@@ -91,17 +145,17 @@ func importWithSrcDir(path string, srcDir string, mode build.ImportMode, install
 	case "crypto/x509", "os/user":
 		// These stdlib packages have cgo and non-cgo versions (via build tags); we want the latter.
 		bctx.CgoEnabled = false
+	case "github.com/gopherjs/gopherjs/js", "github.com/gopherjs/gopherjs/nosync":
```

Comment:
Isn't it ok to check in the above way like `strings.HasPrefix(path, gopherJSRoot+string(filepath.Separator)` for consistency?

Reply:
OK, thanks. I feel like this set is duplicated with `FS` definition in `fs.go`. Would it be possible to unify these sets into one?

Reply:
The "core" GopherJS packages are enumerated and documented in the package comment of `./compiler/gopherjspkg` package (see `compiler/gopherjspkg/doc.go` file).

Here, the `./build` package implements that. It uses import paths.

In `gopherjspkg.FS`, the code picks up those packages via relative directory paths.

I don't think anything more needs to be done, this is very simple, and trying to make it more DRY isn't worth the effort (I'm usually all for it, but it needs to be worthwhile).